src/adapters/RoomsAdapter.test.js

```diff
@@ -49,6 +35,21 @@ describe('Rooms Adapter Interface', () => {
     );
   });
 
+  test('getPreviousRoomActivities() returns a promise rejection since it is not defined', async () => {
+    try {
+      await roomsAdapter.getPreviousRoomActivities('ID');
+    } catch (error) {
+      expect(error.message).toEqual('getPreviousRoomActivities(ID) must be defined in RoomsAdapter');
+    }
+  });
+
+  test('hasMoreActivities() throws error since it is not defined', () => {
+    try {
```

Comment:
Does this need to be in a try/catch?

Reply:
Yes, otherwise it literally errors out the process.